content/docs/contributing/to-docs/overview.md

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
This is not a new doc. This doc used to be `get-started` and it was renamed to `overview.md`